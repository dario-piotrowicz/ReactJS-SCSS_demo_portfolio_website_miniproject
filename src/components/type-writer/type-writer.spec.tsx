import React from 'react';
import { mount } from 'enzyme';
import TypeWriter from './type-writer.component';

describe('testimonials section component', () => {
  it('should match its snapshot', () => {
    const params = {
      leadingFixedText: 'I am ',
      waitTime: 1000,
      phrases: ['a test'],
    };
    expect(mount(<TypeWriter {...params} />)).toMatchSnapshot();
  });
});
