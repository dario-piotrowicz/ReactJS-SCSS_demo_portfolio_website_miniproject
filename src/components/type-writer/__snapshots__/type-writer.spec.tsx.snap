// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testimonials section component should match its snapshot 1`] = `
<TypeWriter
  leadingFixedText="I am "
  phrases={
    Array [
      "a test",
    ]
  }
  waitTime={1000}
>
  <div
    className="type-writer"
  >
    <h1>
      <span
        className="text"
      >
        I am 
         
      </span>
    </h1>
  </div>
</TypeWriter>
`;
